assembly/server-legacy/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>pdi-dataservice-client-assemblies</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pentaho-dataservice-server-legacy</artifactId>
+  <packaging>feature</packaging>
+  <name>Pentaho Community Edition Project: ${project.artifactId}</name>
+  <description>Pentaho Data Services Client for Server - Legacy Plugins Compatibility</description>
+
+  <dependencies>
+    <!--
+        This bundles is needed because we must inject the API on the the server's karaf custom.properties, to then
```

Comment:
Which `bundles`? are you mentioning `server-legacy` itself? If so put the commentary outside of `<dependencies>`. Are you refering to some / all artifacts in  `<dependencies>`? If so, state it explicitly.

Reply:
I'm referring the bundle itself, so I will move the comment up after </parent>